### ISSUE TITLE
In resource view the resource category lacks context for a screen reader

### ISSUE THREAD

**Issue by jhung** [platform-coop-toolkit/coop-library#136]
**Describe the bug**
When navigating a single resource, the resource category is communicated via a screen reader without much context.

For example, if viewing the video resource "OPEN 2017 – Open + Co op: A force to be reckoned with?",  NVDA will report: "Main landmark video heading level 1 open two-thousand and seventeen ...". In this case there is no indication that "video" is a category.

**To Reproduce**
1. Launch a screen reader (i.e. NVDA)
2. Go to the single resource 'OPEN 2017 – Open + Co op: A force to be reckoned with?'
3. Listen to the screen reader report back the contents of the page. NVDA will report: "Main landmark video heading level 1 open two-thousand and seventeen ...". In this case there is no indication that "video" is a category.

**Expected behavior**
The category name should be prefixed with screen reader accessible text "Resource category: " or "Resource categorized under ...".

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox
 - Version 72.0.1
